althea_kernel_interface/src/exit_server_tunnel.rs

```diff
@@ -89,6 +102,64 @@ impl dyn KernelInterface {
             ],
         )?;
 
+        // Set up link local mesh ip in wg_exit as fe80 + rest of mesh ip of exit
+        if let IpAddr::V6(ip6) = exit_mesh {
```

Comment:
the first, second, and third segments must be zero, please use this functoin here rather than re-implementing it

https://github.com/althea-net/althea_rs/blob/master/althea_kernel_interface/src/open_tunnel.rs#L6